### PR TITLE
Optional cbindgen dependency

### LINKED ISSUE #20: Would you consider making cbindgen dependency optional?
I have a pure rust project which has no need for the generated bindings. 

I'm not familiar enough with cbindgen to know what the conventions are, but I was thinking it might make sense to simply make cbindgen an optional build dependency, and only generate the bindings when the feature is enabled.

Are you amenable to a PR?

For context, what brought me here was I noticed that your (up-to-date!) cbindgen resulted in my project duplicating the dependency. Now, I know I should go hound all the other crates to update their cbindgen, but the only thing better than an up-to-date dependency is no dependency at all!

### PR DESCRIPTION
**Pull request by michaelkirk**
FIXES #20

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -16,13 +16,13 @@ num-traits = "0.2.8"
 thiserror = "1.0.4"
 
 [build-dependencies]
-cbindgen = "0.22.0"
+cbindgen = { version = "0.22.0", default-features=false, optional = true}
```
Comment (michaelkirk) [inline Cargo.toml:19]: note also the `default-features=false`, which avoids building `clap` in this case. 

I'm trying to get better about disabling unnecessary features, but it's hard to remember.

**Comment by urschrei** [comment]
Absolutely no problem.
bors r+

**Comment by urschrei** [comment]
(Does this repo have bors?)

**Comment by urschrei** [comment]
Released as [2.4.1](https://crates.io/crates/polylabel/2.4.1)!

**Pull request merged**